omero/sysadmins/unix/install-web.txt

```diff
@@ -86,28 +68,7 @@ configuration redirect the output of the following command into a file:
 
     $ bin/omero web config apache
 
-::
-
-    ...
-    ###
-    ### Stanza for OMERO.web created 2012-07-12 16:44:16.112099
-    ###
-    FastCGIExternalServer "/usr/local/dev/openmicroscopy/dist/var/omero.fcgi" -host 0.0.0.0:4080 -idle-timeout 60
-
-    <Directory "/usr/local/dev/openmicroscopy/dist/var">
-        Options -Indexes FollowSymLinks
-        Order allow,deny
-        Allow from all
-    </Directory>
-
-    <Directory "/usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static">
-        Options -Indexes FollowSymLinks
-        Order allow,deny
-        Allow from all
-    </Directory>
-
-    Alias /omero/static /usr/local/dev/openmicroscopy/dist/lib/python/omeroweb/static
-    Alias /omero "/usr/local/dev/openmicroscopy/dist/var/omero.fcgi/"
+.. literalinclude:: apache-omero.conf
 
```

Comment:
This is much improved WRT being up-to-date and using sensible paths.  Just wondering if we could get away without having it in the documentation at all and just have the instructions to generate it, which removes the need to keep it up to date?